cinema/models.py

```diff
@@ -41,6 +45,7 @@ class Movie(models.Model):
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to="media/uploads/movies/images")
```

Comment:
Image format of saving must be next: f"{slugify(movie.title)}-{uuid}.{ext}"

Reply:
DRF does it under under the hood